lms/static/sass/elements/_controls.scss

```diff
@@ -285,6 +285,46 @@
   }
 }
 
+// imitating the pattern library
+// starts with overrides
+%btn-pl-default-base {
+  @include box-sizing(border-box);
+  @extend %t-copy-base;
+  letter-spacing: 0;
```

Comment:
Thanks for the 411. Nahh, a inline comment like the following would be great though.

```scss
%btn-pl-default-base {
  letter-spacing: 0; // it resets the letter-spacing or it gets the hose + inherited values from crappy older rules
```